src/styles/styles.js

```diff
@@ -3249,6 +3250,59 @@ const styles = (theme) => ({
         lineHeight: variables.lineHeightXXLarge,
     },
 
+    eReceiptAmount: {
+        ...headlineFont,
+        fontSize: variables.fontSizeXXXLarge,
+        lineHeight: variables.lineHeightXXXLarge,
+        color: colors.green400,
```

Comment:
I discussed with @shawnborton and the rest of the design team that it's ok to use a specific color here, since this is a style that will be the same across all themes.

Reply:
Can we still define it in the themes file? importing `colors.js` into styles.js  in the past resulted in a lot of contributors using incorrect colors

Reply:
You can see how we use CONST colors for the animated page headers i.e
[SCREENS.SETTINGS.WORKSPACES]: colors.pink800,